paasta_tools/kubernetes/bin/kubernetes_remove_evicted_pods.py

```diff
@@ -148,7 +148,7 @@ def main() -> None:
     kube_client = KubeClient()
 
     evicted_pods = evicted_pods_per_service(kube_client)
-    notify_service_owners(evicted_pods, args.soa_dir, args.dry_run)
+    # notify_service_owners(evicted_pods, args.soa_dir, args.dry_run)
```

Comment:
Could you add a comment which links to ticket number so we won't forget why we do this.

Reply:
can you just delete the line rather than commenting it out?